src/zenml/zen_stores/sql_zen_store.py

```diff
@@ -6560,14 +6563,17 @@ def create_tag_resource(
     def delete_tag_resource(
```

Comment:
Is this method and the other still used now that the interface no longer exists?

Reply:
Yeah, called inside helpers.